### PR TITLE
Attempt to fix mobile for onboard page

### PR DESCRIPTION
**Pull request by maxwofford**

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/onboard**](https://site-e4ks9rdft.hackclub.dev/onboard) |
|:-:|
|<a href="https://site-e4ks9rdft.hackclub.dev/onboard">Screenshot of /onboard</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/IUZClDGnSyVyn2ouHAjaPm4i3Rp7Sp56BhX2DlxOiPBi8EUjlJx8rMatfVSZv5i57I-lP4ZuGH7AIaTMvyHUdA.png">(view full size)</a> |







Commit 74f5b2de9effa3e9fa99d2a6e9e40a477e9dcd38 (https://site-e4ks9rdft.hackclub.dev).

File: components/onboard/youtube-video.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
   title="YouTube video player",
+  width=null,
+  height=null,
 }) => {
-  const src = `https://www.youtube.com/embed/${youtubeID}`
+  const src = new URL(`https://www.youtube.com/embed/${youtubeID}`)
+
+  if (list) {
+    src.searchParams.set('list', list)
+  }
```
Comment (cjdenio) [inline components/onboard/youtube-video.js]:
```suggestion
  const src = useMemo(() => {
    const url = new URL(`https://www.youtube.com/embed/${youtubeID}`)
    if (list) {
      url.searchParams.set('list', list)
    }
    return url
  }, [youtubeID, list])
```

File: components/onboard/youtube-video.js [hunk 2]
```diff
@@ -1,10 +1,15 @@
 const YoutubeVideo = ({
-  width=560,
-  height=315,
   'youtube-id': youtubeID = 'dQw4w9WgXcQ',
+  list=null,
   title="YouTube video player",
+  width=null,
+  height=null,
```
Comment (cjdenio) [inline components/onboard/youtube-video.js]:
```suggestion
  list,
  title = "YouTube video player",
  width,
  height,
```

Max.. you write too much Ruby

Comment (maxwofford) [reply components/onboard/youtube-video.js]: called out– I actually typed `nil` on these lines before correcting myself.

Comment (cjdenio) [reply components/onboard/youtube-video.js]: 😂 I know the feeling

**Pull request merged**